components/Layout/Layout.tsx

```diff
@@ -6,11 +6,75 @@ import FooterSlackCTA from './FooterSlackCTA';
 import Footer from './Footer';
 import ThemeChanger from './ThemeChanger';
 
-const Layout = ({ title, children, lastPublishedAt }: LayoutProps) => (
+const Layout = ({ title, meta, lastPublishedAt, children }: LayoutProps) => (
     <>
         <Head>
             <title>{title}</title>
-            <link rel="icon" href="/favicon.ico" />
+            <meta
+                name="description"
+                content={meta.searchDescription as string}
+            />
+            <meta
+                name="viewport"
+                content="width=device-width, initial-scale=1"
+            />
+
+            <meta property="og:type" content="website" />
+            <meta
+                property="og:url"
+                content={`https://areweheadlessyet.wagtail.org/${
+                    meta.slug !== 'are-we-headless-yet' ? meta.slug : ''
+                }`}
+            />
+            <meta
+                property="og:title"
+                content={meta.seoTitle ? (meta.seoTitle as string) : title}
+            />
+            <meta property="og:image" content={meta.socialImageUrl as string} />
```

Comment:
Please could we also add support for Twitter here: open graph covers pretty much everything else 👍

```
<meta property="twitter:card" content="summary_large_image">
<meta property="twitter:url" content="...">
<meta property="twitter:title" content="...">
<meta property="twitter:description" content="...">
<meta property="twitter:image" content="...">

```

Reply:
Added.